From 7715b8d4b833dfe3d997f21c8055d33975ed84c8 Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Wed, 29 Jun 2016 16:03:33 +0530
Subject: [PATCH 68/93] driver: pfe_eth: use #ifdef instead of #if

There is no need of checking the value of DEBUG,
so use #ifdef preprocessor directive instead of #if.

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
---
 drivers/net/pfe_eth/pfe_driver.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/pfe_eth/pfe_driver.c b/drivers/net/pfe_eth/pfe_driver.c
index 6774fb4..a3a3e49 100644
--- a/drivers/net/pfe_eth/pfe_driver.c
+++ b/drivers/net/pfe_eth/pfe_driver.c
@@ -50,7 +50,7 @@ int pfe_recv(unsigned int *pkt_ptr, int *phy_port)
 	dprint("Pkt recv'd: Pkt ptr(%p), len(%d), gemac_port(%d) status(%08x)\n",
 				hif_header, len, hif_header->port_no, bd->status);
 
-#if DEBUG
+#ifdef DEBUG
 	{
 		int i;
 		unsigned char *p = (unsigned char *)hif_header;
-- 
1.7.9.5

